### PR TITLE
Enhance inspector

### PR DESCRIPTION
**Pull request by TatriX**
See #61

### CODE REVIEW

**Comment by TatriX** [comment]
Some examples:
```js
let foo = {bar: "Hello baz!"};
foo;

/* * * * * */

Object
  bar: "Hello baz!"

  __defineGetter__: function {  }
  __defineSetter__: function {  }
  __lookupGetter__: function {  }
  __lookupSetter__: function {  }
  __proto__: function {  }
  constructor: function {  }
  hasOwnProperty: function {  }
  isPrototypeOf: function {  }
  propertyIsEnumerable: function {  }
  toLocaleString: function {  }
  toString: function {  }
  valueOf: function {  }

```

```js
class Boo {
    constructor() {
        this.hi = "there";
    }
}
let bar = new Boo();
bar;

/* * * * * */

Boo

  constructor: function {  }
  hi: "there"

  __defineGetter__: function {  }
  __defineSetter__: function {  }
  __lookupGetter__: function {  }
  __lookupSetter__: function {  }
  __proto__: function {  }
  hasOwnProperty: function {  }
  isPrototypeOf: function {  }
  propertyIsEnumerable: function {  }
  toLocaleString: function {  }
  toString: function {  }
  valueOf: function {  }

```

**Comment by NicolasPetton** [comment]
Thanks for the PR!

Why so many commits?

File: indium-inspector.el [hunk 1]
```diff
@@ -55,9 +55,24 @@
...
+  (seq-reduce (lambda (result property)
+                (push property
+                      (if (string-match-p "{ \\[native code\\] }$"
+                                          (map-nested-elt property '(value description)))
+                          (car result)
+                        (cadr result)))
+                result)
+              properties
+              (list nil nil)))
+
```
Comment (NicolasPetton) [inline indium-inspector.el]: Could you add a unit test for this?

**Comment by NicolasPetton** [comment]
Awesome, thanks!

**Pull request merged**